README.md

```diff
@@ -51,11 +51,13 @@ $ jiraprinter --help
 
   Options:
 
-    -h, --help               output usage information
-    -V, --version            output the version number
-    -u, --user [username]    The JIRA username ($JIRA_USER)
-    --password [password]    The JIRA password ($JIRA_PASS)
-    -h, --host [host]        The JIRA hostname ($JIRA_HOST)
+    -h, --help                                  output usage information
+    -V, --version                               output the version number
+    -u, --user [username]                       The JIRA username ($JIRA_USER)
+    --password [password]                       The JIRA password ($JIRA_PASS)
+    -h, --host [host]                           The JIRA hostname ($JIRA_HOST)
+    -pqr --printQR [printQR]                    Whether to include an issue link QR code in the print out ($PRINT_QR)
```

Comment:
I'm not really a fan of multi-character short options... How about just having a long option like `--disable-qr` (also, I'd rather this be true by default so it's consistent with previous behaviour)